src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -371,6 +372,43 @@ else if (!vc.getAlternateAllele(0).equals(Allele.NON_REF_ALLELE)){      //there
         }
     }
 
+    /*
+     * Test that in VCF mode we're consistent with past GATK4 results
+     */
+    @Test(dataProvider="HaplotypeCallerTestInputs")
+    public void testFloorGVCFBlocks(final String inputFileName, final String referenceFileName) throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File output = createTempFile("testFloorGVCFBlocks", ".vcf");
+
+        final List<String> requestedGqBands = Arrays.asList("10","20","30","40","50","60");
+
+        final ArgumentsBuilder args = new ArgumentsBuilder().addInput(new File(inputFileName))
+                .addReference(new File(referenceFileName))
+                .addInterval(new SimpleInterval("20:10000000-10100000"))
```

Comment:
This interval might be unnecessarily large for this test -- could you shrink it down?